ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/nodes/pairs/TomlKeyValue.kt

```diff
@@ -138,13 +138,7 @@ public fun String.parseValue(lineNo: Int, config: TomlInputConfig): TomlValue =
     "true", "false" -> TomlBoolean(this, lineNo)
     // ===== strings
     else -> when (this.first()) {
-        '\'' -> if (this.startsWith("'''")) {
-            // actually the support for multiline strings will be here, but now just a simple mock
-            // this mock will raise an exception here
-            TomlBasicString(this, lineNo)
-        } else {
-            TomlLiteralString(this, lineNo, config)
-        }
+        '\'' -> TomlLiteralString(this, lineNo, config)
         // ===== basic strings
         '\"' -> TomlBasicString(this, lineNo)
```

Comment:
So we received a multiline string and are parsing it with `TomlBasicString`? 
It's ok, if it's not a mistake, but while reading the code - I would expect here different types of strings that start from `"`: Basic, Multiline, etc. 

As this code was intended to be an entry point for determination of different types. So I got a little bit confused.